setup.py

```diff
@@ -43,7 +43,7 @@ def run_tests(self):
 
 requires = [
     'chardet>=3.0.2,<5',
-    'idna>=2.5,<3',
+    'idna>=2.5,<4',
```

Comment:
that should be good for both Python 2 and 3:
```suggestion
    'idna>=2.5,<3 ; python_version < "3"',
    'idna>=2.5,<4 ; python_version >= "3"',
```